### PR TITLE
[ICDS Dashboard]: Adding PSE Eligible cond. to the lunch_count

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
##### SUMMARY
https://dimagi-dev.atlassian.net/browse/ICDS-1588

### CODE REVIEW

**Comment by nitishdubedimagi** [comment]
@ShubhamBansal1997 please ping @calellowitz separately as well since he may be too occupied with Covid work

File: custom/icds_reports/tests/agg_tests/test_export_data.py [hunk 1]
```diff
@@ -3366,8 +3366,8 @@ def test_growth_tracker_export(self):
                       None,
                       None,
                       'N/A',
-                      0,
-                      0,
+                      None,
```
Comment (calellowitz) [inline custom/icds_reports/tests/agg_tests/test_export_data.py:3369]: Why do these go from 0 to `None` and what does that change in the output?

**Review by calellowitz — APPROVED** [review]
curious how this changes the output but otherwise seems fine

File: custom/icds_reports/tests/agg_tests/test_export_data.py (hunk 1 above)
Comment (ShubhamBansal1997) [reply custom/icds_reports/tests/agg_tests/test_export_data.py:3369]: If the beneficiary is not pse_eligible we put his/her lunch count as NULL which is not present earlier.

**Pull request merged**